tests/visualization_tests/test_timeline.py

```diff
@@ -0,0 +1,62 @@
+from io import BytesIO
+from typing import List
+
+import pytest
+
+import optuna
+from optuna.study.study import Study
+from optuna.trial import FrozenTrial
+from optuna.trial import TrialState
+from optuna.visualization._plotly_imports import go
+import optuna.visualization._timeline
+from optuna.visualization._timeline import _get_timeline_info
+
+
+@pytest.mark.parametrize(
+    "n",
+    [0, 3],
+)
+def test_get_timeline_info(n: int) -> None:
```

Comment:
How about testing the case when
(1) `trial.datetime_start` is None
(2) `trial.datetime_complete` is None
(3) Both of above

Reply:
Agreed. I revised the code in https://github.com/optuna/optuna/commit/d4e07207df3079fa308987057d6df2dc7ff29492 . 
According to the implementation of [`create_trial`](https://github.com/optuna/optuna/blob/master/optuna/trial/_frozen.py#L558-L566) , there is no `TrialState` such that `trial.datetime_start` is None and `trial.datetime_complete` is not None.